packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -89,8 +89,8 @@ class Authenticator extends StatefulWidget {
     ConfirmSignInMFAForm? confirmSignInMFAForm,
     this.stringResolver = const AuthStringResolver(),
     required this.child,
-  })  : useAmplifyTheme = false,
-        super(key: key) {
+    this.useAmplifyTheme = true,
```

Comment:
IMO this should still default to false. The standard material theme is going to feel _much_ more familiar to customers than the amplify theme. Also, if customers drop this into an existing MaterialApp that has the `theme` attribute set, the authenticator will pick up their theme automatically if this is false, which would be pretty nice.

On a related note, the tabs on the authenticator don't look very good if the user sets this to false. 

![Screen Shot 2021-11-11 at 12 39 56 PM](https://user-images.githubusercontent.com/20613561/141344934-9ebea0d3-265e-4f43-ab2c-6a5abcadc7ab.png)